avidemux_fr.ts

```diff
@@ -1153,12 +1153,12 @@ Ignorer les informations de timing ?</translation>
     <message>
         <location line="+1"/>
         <source>_Revert to saved default output settings on video load</source>
-        <translation>_Revenir aux paramètres par défaut enregistrés lors du chargement vidéo</translation>
+        <translation>Retour paramètres de sortie par défaut au chargement de vidéo</translation>
```

Comment:
Last time I forgot to submit the comment, losing it, oops.

If I rephrase the source string as "Restore custom output parameters, saved as default, when loading a video", does the fixed translation still convey its meaning?